Sources/SwiftCurrent_Testing/CustomAssertions.swift

```diff
@@ -18,7 +18,16 @@ import UIKit
 #if !os(watchOS)
 /// Assert that a workflow was launched and matches the workflow passed in
 public func XCTAssertWorkflowLaunched<F>(from VC: UIViewController, workflow: Workflow<F>, file: StaticString = #filePath, line: UInt = #line) {
+    XCTAssertWorkflowLaunched(from: VC, workflow: AnyWorkflow(workflow), file: file, line: line)
+}
+
+/// Assert that a workflow was launched and matches the workflow passed in
+public func XCTAssertWorkflowLaunched(from VC: UIViewController, workflow: AnyWorkflow?, file: StaticString = #filePath, line: UInt = #line) {
     let last = VC.launchedWorkflows.last
+    guard let workflow = workflow else {
+        XCTAssertNil(last, "workflow found when none expected", file: file, line: line)
```

Comment:
If we're going to allow people to assert no workflow was launched I think it should be a separate method, otherwise this looks great!

Reply:
Fair enough, although the workflow here is only optional as a convience for the case where data-driven workflow generation fails and comes back nil. If your test is generating a nil expected workflow, you'd hope no actual workflow launched.